extensions/css-language-features/package.json

```diff
@@ -356,6 +360,28 @@
             "scope": "resource",
             "default": null,
             "markdownDescription": "%css.format.maxPreserveNewLines.desc%"
+          },
+          "css.format.formatPastedFiles": {
+            "type": "boolean",
+            "scope": "resource",
+            "default": false,
+            "markdownDescription": "%css.format.formatPastedFiles.desc%"
+          },
+          "css.format.formatDroppedFiles": {
+            "type": "boolean",
+            "scope": "resource",
+            "default": false,
+            "markdownDescription": "%css.format.formatDroppedFiles.desc%"
+          },
+          "css.format.formattedFileQuoteType":{
```

Comment:
For this, maybe `css.preferredQuoteType`? Being under `format` suggest that running format document would change existing quotes in the file